src/components/AdoptionProcess/ProcessContainer.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import ProcessItems from "./ProcessItems";
+import { Container, Row, Col } from "react-bootstrap";
+import "bootstrap/dist/css/bootstrap.min.css";
+
+const ProcessContainer = ({ adoptionProcess, sectionTitle }) => {
+  return (
+    <Container>
+      <Row className="adoptionContainer py-5">
+        <Col sm={12}>
+          <h1>{sectionTitle.title}</h1>
+          <Row className="mt-5 items">
+            {adoptionProcess &&
```

Comment:
please choose a better variable name that describes what kind of data does the variable hold

Reply:
Hi @halitbatur , I changed my variable names for data and title. I m not able to request re-view since there is a yellow button next to your name saying "awaiting approval from halit batur". And also I attach my component's visuals for how it looks like on browser according your request. ( desktop and mobile version ) 
PTLA

![image](https://user-images.githubusercontent.com/65858467/129446075-2d7ab8b6-33b0-4768-917a-a328c2aa5a8a.png)

![image](https://user-images.githubusercontent.com/65858467/129446096-83a19af2-0e01-4331-ab84-e2b9f8200656.png)